coalahtml/_coalahtml/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "devDependencies": {
+    "angular-mocks": "^1.5.0",
+    "chai": "^3.5.0",
+    "karma": "^0.13.21",
```

Comment:
Yes. I have removed all the tests outside keeping only bower configs intact. Sending PR :+1: